packages/workbox-build/src/lib/utils/warn-about-config.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const logHelper = require('../log-helper');
+
+/**
+ * A helper to find and log any configuration options that were set which are
+ * not valid for a given usage mode.
+ *
+ * @param {Array<String>} blacklist The names of invalid configuration options.
+ * @param {Object<String,String>} fullConfig The configuration options to check.
+ */
+module.exports = (blacklist, fullConfig) => {
+  const invalidConfig = Object.keys(fullConfig)
```

Comment:
Nit: Rename to invalidConfigKeys (Just reads better for if (invalidConfigKeys.length > 0)

Reply:
Done.